app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -26,11 +26,12 @@ def tab(*args)
         destination_url = options[:url] || spree.public_send("#{options[:route]}_path")
         titleized_label = Spree.t(options[:label],
                                   default: options[:label],
-                                  scope: [:admin, :tab]).titleize
+                                  scope: [:admin, :tab]).capitalize
 
         css_classes = []
 
-        if options[:icon]
+        if options[:icon] && !OpenFoodNetwork::FeatureToggle.enabled?(:admin_style_v3,
+                                                                      spree_current_user)
```

Comment:
You can use an application helper here:
```suggestion
        if options[:icon] && !feature?(:admin_style_v3, spree_current_user)
```